source/users.rst

```diff
@@ -0,0 +1,325 @@
+
+  {
+    "display_name": "User One",
+    "username": "user1",
+    "email": "user1@example.org",
+    "site_spectator": true,
+    "site_manager": false,
+    "site_admin": false,
+    "created_at": "2016-02-15",
+    "updated_at": "2016-02-15",
+    "deleted_at": "2017-06-21",
+    "active": false,
+    "meta": "extra metadata about user"
+  }
+
+POST /users
+~~~~~~~~~~~
+
+Create a new user.
+
+Request:
+
+.. code-block:: javascript
+
+  {
+    "displayname": "X. Ample User",
+    "username": "example",
+    "password": "password",
+    "email": "example@example.com"
+    "site_spectator": true,
+    "site_manager": false,
+    "site_admin": false,
+    "active": true,
+    "meta": "Some metadata about the user"
+  }
+
+Response:
+
+.. code-block:: javascript
+
+  {
+    "displayname": "X. Ample User",
+    "username": "example",
+    "email": "example@example.com"
+    "site_spectator": true,
+    "site_manager": false,
+    "site_admin": false,
+    "active": true,
+    "created_at": "2016-02-15",
+    "updated_at": "2016-02-15",
+    "deleted_at": null,
+    "active": true,
+    "meta": "Some metadata about the user"
+  }
+
+.. caution::
+
+  It is the client's responsibility to hash the password before sending it to
+  this endpoint, unlike the /login endpoint (see :ref:`auth<auth>`). The
+  password should be hashed with bcrypt, using 10 rounds. The bcrypt prefix
+  should be "2a" (different implementations may use different prefixes, but the
+  API requires consistency for authentication).
+
+.. note::
+
+  This endpoint may only be accessed by admins and sitewide managers.
+
+.. note::
+
+  It is recommended that admins provide the user with a temporary password
+  and have the user change the password when they log in.
+
+~~~~~~~~~~~~~~~~~~~~~
+
+POST /users/:username
+~~~~~~~~~~~~~~~~~~~~~
+
+Original object:
+
+.. code-block:: javascript
+
+  {
+    "display_name": "User One",
+    "username": "user1",
+    "email": "user1@example.org",
+    "site_spectator": true,
+    "site_manager": false,
+    "site_admin": false,
+    "active": true,
+    "created_at": "2016-02-15",
+    "updated_at": "2016-02-15",
+    "deleted_at": null,
+    "active": false,
+    "meta": "extra metadata about user"
+  }
+
+Request body (made by a ``site_admin`` user):
+
+.. code-block:: javascript
+
+  {
+    "display_name": "New Displayname",
+    "password": "Battery Staple",
+    "email": "user1+new@example.org",
+    "meta": "Different metadata about user1",
+    "site_spectator": true,
+    "site_manager": true,
+    "site_admin": false,
+  }
+
+The response will be:
+
+.. code-block:: javascript
+
+  {
+    "display_name": "New Displayname",
+    "username": "user1",
+    "email": "user1+new@example.org",
+    "site_spectator": true,
+    "site_manager": true,
+    "site_admin": false,
+    "created_at": "2016-02-15",
+    "updated_at": "2016-02-18",
+    "deleted_at": null,
+    "meta": "Different metadata about user1"
+  }
+
+.. caution::
```

Comment:
One is in the "creating users" section, the other's in the "updating users" section. So we copy it so that users who just want to read about how to update a user still get the warning.